### PR TITLE
Add link to test helper

### PR DESCRIPTION
**Pull request by w3cj**
This adds a link to the test helper in the testing guide. I didn't find the helper until after using `app.request` for a few tests.

### CODE REVIEW

File: docs/guides/testing.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ In this section, let's test with Cloudflare Workers and Jest.
 All you do is create a Request and pass it to the Hono application to validate the Response.
 And, you can use `app.request` the useful method.
 
+> For a typed test client see the [testing helper](/docs/helpers/testing).
```
Comment (yusukebe) [inline docs/guides/testing.md:13]: How about using the `Tip` box?

https://vitepress.dev/guide/markdown#default-title

CleanShot 2024-08-19 at 09 13 34@2x

**Comment by yusukebe** [comment]
Hi @w3cj 

Adding the link is good! I've left a comment. Thanks!

**Comment by w3cj** [comment]
Good idea! Updated to use tip.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@w3cj 

Great! Thank you for your contribution!

**Pull request merged**